views/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Bowling Score Sheet</title>
+<script src="../jquery.js"></script>
+<link rel="stylesheet" href="../public/index.css">
+<script src="../src/game.js"></script>
+<script src="../src/scoreCalculator.js"></script>
+<script src="../src/interface.js"></script>
+  </head>
+  <body>
+    <header>
+      <h1> Bowling Scoresheet <h1>
+    </header>
+
+
+    <div id='scorecard'>
+      <img class='strike' src="../public/Strike.gif">
+
+  <table id='scorecardTable' class='scorecard' cellpadding='1' cellspacing='0'>
```

Comment:
It's possible to build this table using js, leaving you with fewer lines of HTML to write :)

Reply:
Hmm... I imagine there is a way of writing javascript code to do lots of HTML things for you, but haven't seen/ used this yet